content/en/docs/tasks/debug/debug-cluster/troubleshooting-kubectl-connectivity.md

```diff
@@ -0,0 +1,118 @@
+---
+reviewers:
+# Should be added by an experienced contributor
+title: "Troubleshooting kubectl Connectivity"
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+This documentation is about investigating and diagnosing connectivity issues between
+{{<glossary_tooltip text="kubectl" term_id="kubectl">}} and your Kubernetes cluster.
+If you are unable to connect to your cluster using kubectl, this document describes
+several common scenarios and potential solutions to identify and address the likely
+cause.
+
+<!-- body -->
+
+## {{% heading "prerequisites" %}}
+
+* Kubernetes cluster is installed.
+* `kubectl` is installed and configured to communicate with the cluster.
```

Comment:
If that prerequisite is met, the user has little to do. Consider rephrasing.

Reply:
Sure!